plugins/crashes/frontend/public/javascripts/countly.views.js

```diff
@@ -850,8 +854,21 @@
                     this.crashesBeingSymbolicated.push(crash._id);
                     this.$store.dispatch("countlyCrashes/crashgroup/symbolicate", crash)
                         .then(function() {
+                            CountlyHelpers.notify({
+                                title: jQuery.i18n.map["crash_symbolication.symbolication-success"],
```

Comment:
We can avoid jQuery here and  use `CV.i18n`

Reply:
Updated